### PR TITLE
Consider adding "Move +" and "Move -" KSPEvents for better KOS integration.

### PR DESCRIPTION
**Pull request by ZiwKerman**
Recent KOS career limits changes prohibit use of KSPActions in career mode until VAB/SPH is fully upgraded. In order to provide access to IR part moving through KOS earlier in the career mode I decided to create very simple KSPEvents "Move +" and "Move -" with the same functionality as the corresponding KSPActions.

### CODE REVIEW

**Comment by Dunbaratu** [comment]
It's not just for kOS.  It's better integrated into how the rest of KSP's interface itself works this way.  The main stock KSP game trains users to think that "I wan't to do a thing with this one part" means "I need to right click this one part and see what's on its menu".

File: .gitignore [hunk 1]
```diff
@@ -11,4 +11,5 @@ x64/
 build/
 bld/
 [Bb]in/
-[Oo]bj/
\ No newline at end of file
+[Oo]bj/
+InfernalRobotics/InfernalRobotics/InfernalRobotics.csproj
```
Comment (erendrake) [inline .gitignore]: you might not want to always ignore changes to this file.

Comment (ZiwKerman) [inline .gitignore]: Oh, I accidentally included .gitignore in commit. What is the easiest way to fix that?

**Comment by ZiwKerman** [comment]
Obsolete

**Pull request closed without merging**